src/app/search/basic-search/basic-search.component.html

```diff
@@ -1,18 +1,21 @@
 <mat-expansion-panel expanded>
   <mat-expansion-panel-header> Search </mat-expansion-panel-header>
-  <mat-form-field appearance="outline" class="form-field">
+  <mat-form-field appearance="outline" class="search-form-field">
     <input
       matInput
       [matAutocomplete]="auto"
       [formControl]="searchFormControl"
       matTooltip="Find tools and workflows"
       [matTooltipPosition]="'after'"
       list="suggestions"
-      class="w-100 mb-3"
       placeholder="Enter search term"
       type="text"
+      [(ngModel)]="value"
```

Comment:
i prefer the other method without ngModel. ngModel is for two-way binding which isn't needed here

Reply:
Sorry, which other method are you referring to?